<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="2" colspan="4">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>228</width>
        <height>6</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="3" colspan="3">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_ShKey">
       <property name="text">
        <string>移位密钥：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ShKey"/>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QRadioButton" name="encrypt">
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>加密</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="decrypt">
       <property name="text">
        <string>解密</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_run">
       <property name="text">
        <string>运行</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="3" colspan="3">
     <widget class="QLabel" name="label__DeKey">
      <property name="text">
       <string>分解密钥：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>208</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" rowspan="7">
     <widget class="QSplitter" name="splitter_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QLabel" name="label_BeforeDeal">
        <property name="text">
         <string>处理前文本：</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit_BeforeDeal"/>
      </widget>
      <widget class="QSplitter" name="splitter_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QLabel" name="label_AfterDeal">
        <property name="text">
         <string>处理后文本：</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit__AfterDeal"/>
      </widget>
     </widget>
    </item>
    <item row="1" column="1" rowspan="6" colspan="4">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>208</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="5">
     <widget class="QPlainTextEdit" name="plainTextEdit_DeKey"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>456</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
